docs/guides/latest/volumesnapshot/deployment.md

```diff
@@ -0,0 +1,525 @@
+            # name: source-pvc-1-1560662826
+            apiGroup: snapshot.storage.k8s.io
+      - metadata:
+          name: source-pvc-2
+        spec:
+          accessModes: [ "ReadWriteOnce" ]
+          storageClassName: "standard"
+          resources:
+            requests:
+              storage: 6Gi
+          dataSource:
+            kind: VolumeSnapshot
+            name: ${CLAIM_NAME}-1560662826
+            # name: source-pvc-2-1560662826
+            apiGroup: snapshot.storage.k8s.io
+```
+
+Here,
+
+- `spec.target.volumeClaimTemplates`:
+  - `metadata.name` is the name of the restored `PVC` or prefix of the `VolumeSnapshot` name.
+  - `spec.dataSource`: `spec.dataSource` specifies the source of the data from where the newly created PVC will be intialized. It requires following fields to set:
+    - `apiGroup` is the group for resource being referenced. Now, kubernetes supports only `snapshot.storage.k8s.io`.
+    - `kind` is resource of the kind being referenced. Now, kubernetes supports only `VolumeSnapshot`.
+    - `name` is the `VolumeSnapshot` resource name. In `RestoreSession` crd, You can template the name by using the following convention `${CLAIM_NAME}-<backup session creation timestamp in Unix epoch seconds>`, `${CLAIM_NAME}` is resolved by the Stash operator and replaced by the `metadata.name`. You can set the snapshot name directly.
+
+Let's create the `RestoreSession` crd we have shown above.
+
+```console
+$ kubectl create -f ./docs/examples/guides/latest/volumesnapshot/deployment/restoresession.yaml
+restoresession.stash.appscode.com/restore-pvc created
+```
+
+Once, you have created the `RestoreSession` crd, Stash will create a job to restore. We can watch the `RestoreSession` phase to check if the restore process is succeeded or not.
+
+Run the following command to watch RestoreSession phase,
+
+```console
+$ watch -n 1 kubectl get restore -n demo
+Every 1.0s: kubectl get restore -n demo                      suaas-appscode: Tue Jun 18 18:35:41 2019
+
+NAME          REPOSITORY-NAME   PHASE       AGE
+restore-pvc                     Running     10s
+restore-pvc                     Succeeded   1m
+```
+
+So, we can see from the output of the above command that the restore process succeeded.
+
+**Verify Restored PVC :**
+
+Once a restore process is complete, we will see that new PVCs with the name `source-pvc-1` and `source-pvc-2` has been created.
+
+Verify that the PVCs has been created by the following command,
+
+```console
+$ kubectl get pvc -n demo
+NAME           STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+source-pvc-1   Bound    pvc-e32b18f6-91c0-11e9-bd3e-42010a800011   6Gi        RWO            standard       30s
+source-pvc-2   Bound    pvc-e32db936-91c0-11e9-bd3e-42010a800011   6Gi        RWO            standard       30s
+````
+
+Notice the `STATUS` field. `Bound` indicates that PVC has been initialized from the respective VolumeSnapshot.
```

Comment:
`Bound` indicates that PVC has been initialized from the respective VolumeSnapshot.

Is this correct?